src/pages/home/profile/Profile.tsx

```diff
@@ -24,12 +31,30 @@ const Profile: FunctionComponent = () => {
       email,
       firstName,
       genre,
+      id = "",
       lastName,
       nickname,
       pictureUrl,
       role,
+      verifiedStatus,
     } = {},
+    idenfy: { authToken },
   } = useSelector(selectSession);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const isUnverified = verifiedStatus === "unverified";
+  const isPendingVerification = verifiedStatus === "pending";
+  const isVerified = verifiedStatus === "verified";
+
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
+
+  const handleVerificationSession = () => {
+    if (!authToken) {
+      dispatch(requestVerificationToken({ clientId: id }));
```

Comment:
If you are testing this out in your local, I would recommend commenting this line out after you get the authToken and hardcode that in. We have a limited amount of testing tokens and would like to use them once the BE work is done so that is more E2E.